packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v2.3.0-deploy-contract-from-json.test.ts

```diff
@@ -474,22 +455,11 @@ test(testCase, async (t: Test) => {
       gas: 1000000,
       constructorArgs: ["Test Arg"],
     });
-    t2.ok(deployOut, "deployContract() output is truthy OK");
-    t2.ok(
-      deployOut.transactionReceipt,
-      "deployContract() output.transactionReceipt is truthy OK",
-    );
-    t2.ok(
-      deployOut.transactionReceipt.contractAddress,
-      "deployContract() output.transactionReceipt.contractAddress is truthy OK",
-    );
+    expect(deployOut).toBeTruthy();
+    expect(deployOut.transactionReceipt).toBeTruthy();
+    expect(deployOut.transactionReceipt.contractAddress).toBeTruthy();
 
     contractAddress = deployOut.transactionReceipt.contractAddress as string;
-    t2.ok(
-      typeof contractAddress === "string",
-      "contractAddress typeof string OK",
-    );
+    expect(typeof contractAddress).toBeString;
```

Comment:
@awadhana Two things:

1. You are asserting that the `typeof contractAddress` is a string, but that will always be true because the `typeof` keyword always returns a string (it returns the name of the actual type in a string format, like in this case it would literally return "string" indicating that `contractAddress` has a type of string. So you need to remove the `typeof` and that way the assertion will check for the type of `contractAddress` instead of the type of the type of `contractAddress` (very meta, I know)
2. You are referencing the function pointer itself with `toBeString` meaning that you are not actually calling the function and my guess is that nothing actually happens in this line of code. You want to use the parentheses syntax to call function. Whenever in doubt, you can double check this in the IDE directly (thanks to the Typescript type information). See below:
![Screenshot from 2022-01-27 12-58-08](https://user-images.githubusercontent.com/1583231/151443235-e29709a5-4392-468e-89fe-7aa1ce4aff18.png)